src/Router.js

```diff
@@ -14,6 +15,7 @@ const Router = () => {
         <Routes>
           <Route path="/delivery" element={<Delivery />} />
           <Route path="/" element={<Login />} />
+          <Route path="/payComplete" element={<PayComplete />} />
```

Comment:
추가 리뷰를 드리면, 경로는 일반적으로 kebab-case로 작성합니다.
```suggestion
          <Route path="/pay-complete" element={<PayComplete />} />
```